frontend/src/tests/pages/PageNotFound.test.js

```diff
@@ -0,0 +1,71 @@
+import { render, screen, fireEvent, act, waitFor } from "@testing-library/react";
+import PageNotFound from "main/pages/PageNotFound";
+import { QueryClient, QueryClientProvider } from "react-query";
+import { MemoryRouter } from "react-router-dom";
+
+import { apiCurrentUserFixtures }  from "fixtures/currentUserFixtures";
+import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
+import axios from "axios";
+import AxiosMockAdapter from "axios-mock-adapter";
+
+const mockNavigate = jest.fn();
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    useNavigate: () => mockNavigate
+}));
+
+describe("PageNotFound tests", () => {
+
+    const axiosMock =new AxiosMockAdapter(axios);
+    axiosMock.onGet("/api/currentUser").reply(200, apiCurrentUserFixtures.userOnly);
+    axiosMock.onGet("/api/systemInfo").reply(200, systemInfoFixtures.showingNeither);
+
+    const queryClient = new QueryClient();
+    test("renders without crashing", () => {
+        render(
+            <QueryClientProvider client={queryClient}>
+                <MemoryRouter>
+                    <PageNotFound />
+                </MemoryRouter>
+            </QueryClientProvider>
+        );
+    });
+    test("redirects to /movies on submit", async () => {
+
+
+       // mockAdd.mockReturnValue({
+       //     "movie": {
+       //         id: 3,
+       //         name: "Cinderella",
+       //         releasedate: "March 13, 2015",
+       //         director: "Kenneth Branagh"
+       //     }
+       // });
```

Comment:
Please do not leave commented out code in your PRs.

Reply:
I also don't see this commented out code in my local editor for some reason? So I think I will try recommiting these files